src/StagedProposalProcessor.sol

```diff
@@ -312,7 +336,14 @@ contract StagedProposalProcessor is ProposalUpgradeable, PluginUUPSUpgradeable {
                 uint16 newStage = proposal.currentStage + 1;
                 proposal.currentStage = newStage;
 
-                _createPluginProposals(_proposalId, newStage, uint64(block.timestamp));
+                bytes[][] memory params = createProposalParams[_proposalId];
+
+                _createPluginProposals(
+                    _proposalId,
+                    newStage,
+                    uint64(block.timestamp),
+                    params.length > 0 ? params[0] : new bytes[](0)
```

Comment:
Why `params[0]`?
Is this a hardcoded initialization of stage 0?

Reply:
Fixed in https://github.com/aragon/staged-proposal-processor-plugin/pull/8/commits/84d218836b9ff469397fdb86becd19f89dd31598